.travis.yml

```diff
@@ -1,7 +1,8 @@
-sudo: required
+os: linux
+dist: xenial
```

Comment:
This matches what we've got currently, but I wonder if we should move to bionic...

Reply:
Yes, I wanted to add another job with bionic and their default Python but then I thought we specify exact Python version anyway and Python is provided by special means of Travis, so dist should play little to no difference anyway.